### ISSUE TITLE
Center image not centered

### ISSUE THREAD

**Issue by locutus2999** [CSS-Tricks/MovingBoxes#20]
The image in the center is not actually centered. Even if you look on the example the image on the right is cut off more than the image on the left. In my implementation, this anomaly is very obvious. Can someone suggest how to fix this? thanks.

**Comment by Mottie** [comment]
I think the problem is with the way the script calculates the image width and height based on the aspect ratio. I'll look into fixing it, but in the mean time, try adding this CSS:

```
.mb-inside img {
  margin-left: 4px;
}
```

Adjust the margin if you need to, but I think it's about 4 pixels off... at least in the demo page.

**Comment by locutus2999** [comment]
Thanks for the reply Mottie. I am actually not using any images, just text. So I don't think it is because of how it calculates the image size. I've tried playing with all the margin's and I can't get it to correct.

**Comment by Mottie** [comment]
How about trying just the class without the image... if it doesn't work add an "!important" flag after it:

```
.mb-inside {
  margin-left: 4px;
}
```

Sorry I'm really busy this week and don't have time to test it.

**Comment by Mottie** [comment]
Ok this issue has been fixed in v2.0.1... the css modified was:

```
.mb-inside img {
  display: block;
  border: 1px solid #666;
  margin: 0 auto;
  width: 100%;
}
```

Changing the margin to auto fixed it :) Thanks again for bringing up this issue.
